MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,52 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace MagneticFieldParabolicPortable {
+
+    struct Parameters {
```

Comment:
Can you add a comment explaining where these numbers come from ?
For example
```c++
    // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
    // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc .
```

Reply:
Added: https://github.com/cms-sw/cmssw/pull/44360/commits/a088e4686ce71a97ce88a7c87293e4d05854fda8